modules/imap/hm-imap.php

```diff
@@ -526,14 +526,15 @@ public function get_special_use_mailboxes($type=false) {
          * @param bool $lsub flag to limit results to subscribed folders only
          * @return array associative array of folder details
          */
-        public function get_mailbox_list($lsub=false, $mailbox='', $keyword='*') {
+        public function get_mailbox_list($lsub=false, $mailbox='', $keyword='*', $children_capability=true) {
```

Comment:
This method is called in several different places including tests. Can you make sure all of them work with the updated functionality?